File: murmurRelay/app/src/main/java/org/murmurRelay/infrastructures/dtos/ServerDto.java
package org.murmurRelay.infrastructures.dtos;

public class ServerDto {
    private final String domain;
    private final String base64AES;

    public ServerDto(String domain,String base64AES) {
        this.domain = domain;
        this.base64AES = base64AES;
    }

    public String getDomain() {
        return domain;
    }

    public String getBase64AES() {
        return base64AES;
    }
}

File: murmur_relay/src/infrastructures/dto/server_dto.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
pub struct ServerDto{
    domain : String,
    base_64_aes: String,
}

impl ServerDto {
    pub fn new(domain : String, base_64_aes : String) -> Self {
        ServerDto{
            domain,
            base_64_aes,
        }
    }
    
    pub fn domain(&self) -> &String {
        &self.domain
    }
    
    pub fn base_64_aes(&self) -> &String {
        &self.base_64_aes
    }
}